From 3823233773e100874645879ff89c300a31d8e056 Mon Sep 17 00:00:00 2001
From: Gajanan Bhat <gajanan.bhat@intel.com>
Date: Tue, 8 Dec 2015 14:07:06 +0530
Subject: [PATCH 2/6] Revert "SQUASHME! [VPG]: drm/i915: When multiple planes
 are enabled configure pfit to AUTO mode"

This patch reverts the below commit:
	commit ed37dd795d8ce06a493eb7e22541f9f4117445f3
	Author: Arun R Murthy <arun.r.murthy@intel.com>
	Date:   Mon Oct 19 13:33:10 2015 +0530

In kernel we do not restrict mode of panel fitter. All the decision is left to the
hardware composer. Hence reverting restriction in multiplane mode.
The original patch created a regrewssion, panel fitter was not getting
enabled on letterbox/pillarbox mode at all when in full screen video.
Hence reverting the patch.

Change-Id: I956d9269ec6850517daa7718208f9c4746804d46
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-11578
Signed-off-by: Arun R Murthy <arun.r.murthy@intel.com>
Signed-off-by: Gajanan Bhat <gajanan.bhat@intel.com>
Signed-off-by: Vidya Srinivas <vidya.srinivas@intel.com>
Reviewed-on: https://android.intel.com:443/463651
---
 drivers/gpu/drm/i915/intel_display.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index b303643..46f00a1 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -11265,16 +11265,6 @@ int intel_set_disp_calc_flip(struct drm_mode_set_display *disp,
 					(disp->panel_fitter.src_w / disp->panel_fitter.src_h))
 				pfit_mode |= PFIT_SCALING_AUTO;
 
-			/*
-			 * After calculating the aspect ratio and determining the mode
-			 * check for multi-planes. If multi planes panel fitter has to
-			 * be enabled only in AUTO mode.
-			 */
-			if (!dev_priv->maxfifo_enabled) {
-				pfit_mode &= MASK_PFIT_SCALING_MODE;
-				pfit_mode |= PFIT_SCALING_AUTO;
-			}
-
 			/* Enable Panel fitter if any valid mode is set */
 			pfit_control = (1 << 31) | pfit_control;
 			if (disp->panel_fitter.mode == AUTOSCALE) {
-- 
1.9.1

